Nos/Models/Nos.xcdatamodeld/Nos.xcdatamodel/contents

```diff
@@ -2,6 +2,7 @@
 <model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="21513" systemVersion="21G115" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
     <entity name="Author" representedClassName=".Author" syncable="YES" codeGenerationType="category">
         <attribute name="about" optional="YES" attributeType="String"/>
+        <attribute name="display_name" optional="YES" attributeType="String"/>
```

Comment:
any reason this isn't camel case?

Reply:
I was matching the JSON. I think we could get around that by using CodingKeys in the MetadataEventJSON class, then we could use a name without an underscore.